src/Components/HorizontalCard/HorizontalProductCard.jsx

```diff
@@ -96,6 +98,48 @@ function HorizontalProductCard({productDetails})
         } 
     }
 
+    async function addItemToWishlist()
+    {
+        const token=localStorage.getItem('token')
+
+        if(token)
+        {
+            const user = jwt_decode(token)
+            
+            if(!user)
+            {
+                localStorage.removeItem('token')
+                showToast("warning","","Kindly Login")
+                navigate('/login')
+            }
```

Comment:
can't understand why r u doing this

Reply:
It is to check if jwt payload is present after decoding